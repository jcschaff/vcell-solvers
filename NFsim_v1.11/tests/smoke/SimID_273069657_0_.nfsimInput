<sbml><model id="nameless"><ListOfParameters><Parameter id="_F_" type="Constant" value="96485.3321" /><Parameter id="_F_nmol_" type="Constant" value="9.64853321E-5" /><Parameter id="_N_pmol_" type="Constant" value="6.02214179E11" /><Parameter id="_PI_" type="Constant" value="3.141592653589793" /><Parameter id="_R_" type="Constant" value="8314.46261815" /><Parameter id="_T_" type="Constant" value="300.0" /><Parameter id="Kf_r00" type="Constant" value="1.0" /><Parameter id="Kf_r10" type="Constant" value="0.0" /><Parameter id="KMOLE" type="Constant" value="0.001660538783162726" /><Parameter id="Kr_r00" type="Constant" value="2.0" /><Parameter id="Kr_r10" type="Constant" value="0.0" /><Parameter id="s0_Count_init_uM" type="Constant" value="2.0" /><Parameter id="s1_Count_init_uM" type="Constant" value="3.0" /><Parameter id="s2_Count_init_uM" type="Constant" value="4.0" /><Parameter id="s3_Count_init_uM" type="Constant" value="0.0" /><Parameter id="s4_Count_init_uM" type="Constant" value="0.0" /><Parameter id="Size_c0" type="Constant" value="0.5" /><Parameter id="UnitFactor_molecules_uM_neg_1_um_neg_3" type="Constant" value="602.214179" /><Parameter id="UnitFactor_uM_um3_molecules_neg_1" type="Constant" value="0.001660538783162726" /><Parameter id="P_r00_probabilityRate" type="Constant" value="0.003321077566325452" /><Parameter id="P_r00_reverse_probabilityRate" type="Constant" value="2.0" /><Parameter id="P_r10_probabilityRate" type="Constant" value="0.0" /><Parameter id="P_r10_reverse_probabilityRate" type="Constant" value="0.0" /><Parameter id="s0_Count_initCount" type="Constant" value="602.214179" /><Parameter id="s1_Count_initCount" type="Constant" value="903.3212684999999" /><Parameter id="s2_Count_initCount" type="Constant" value="1204.428358" /><Parameter id="s3_Count_initCount" type="Constant" value="0.0" /><Parameter id="s4_Count_initCount" type="Constant" value="0.0" /><Parameter id="K_reserved_0" type="Constant" value="0.003321077566325452"><!--P_r00_probabilityRate--></Parameter><Parameter id="K_reserved_1" type="Constant" value="2.0"><!--P_r00_reverse_probabilityRate--></Parameter><Parameter id="K_reserved_2" type="Constant" value="0.0"><!--P_r10_probabilityRate--></Parameter><Parameter id="K_reserved_3" type="Constant" value="0.0"><!--P_r10_reverse_probabilityRate--></Parameter></ListOfParameters><ListOfMoleculeTypes><MoleculeType id="MT0" hasAnchors="false"><ListOfComponentTypes><ComponentType id="AAA"><ListOfAllowedStates><AllowedState id="c0" /></ListOfAllowedStates></ComponentType><ComponentType id="AAB"><ListOfAllowedStates><AllowedState id="0" /><AllowedState id="1" /></ListOfAllowedStates></ComponentType></ListOfComponentTypes></MoleculeType><MoleculeType id="MT1" hasAnchors="false"><ListOfComponentTypes><ComponentType id="AAA"><ListOfAllowedStates><AllowedState id="c0" /></ListOfAllowedStates></ComponentType><ComponentType id="AAB"><ListOfAllowedStates><AllowedState id="0" /><AllowedState id="1" /></ListOfAllowedStates></ComponentType></ListOfComponentTypes></MoleculeType><MoleculeType id="MT2" hasAnchors="false"><ListOfComponentTypes><ComponentType id="AAA"><ListOfAllowedStates><AllowedState id="c0" /></ListOfAllowedStates></ComponentType><ComponentType id="AAB"><ListOfAllowedStates><AllowedState id="0" /><AllowedState id="1" /></ListOfAllowedStates></ComponentType></ListOfComponentTypes></MoleculeType><MoleculeType id="MT3" hasAnchors="false"><ListOfComponentTypes><ComponentType id="AAA"><ListOfAllowedStates><AllowedState id="c0" /></ListOfAllowedStates></ComponentType><ComponentType id="AAB"><ListOfAllowedStates><AllowedState id="0" /><AllowedState id="1" /></ListOfAllowedStates></ComponentType></ListOfComponentTypes></MoleculeType><MoleculeType id="MT4" hasAnchors="false"><ListOfComponentTypes><ComponentType id="AAA"><ListOfAllowedStates><AllowedState id="c0" /></ListOfAllowedStates></ComponentType><ComponentType id="AAB"><ListOfAllowedStates><AllowedState id="0" /><AllowedState id="1" /></ListOfAllowedStates></ComponentType></ListOfComponentTypes></MoleculeType></ListOfMoleculeTypes><ListOfSpecies><Species id="S1" name="speciesPattern0" concentration="602.214179"><ListOfMolecules><Molecule id="S1_M1" name="MT0"><ListOfComponents><Component id="S1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="S1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Species><Species id="S2" name="speciesPattern1" concentration="903.3212684999999"><ListOfMolecules><Molecule id="S2_M1" name="MT1"><ListOfComponents><Component id="S2_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="S2_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Species><Species id="S3" name="speciesPattern2" concentration="1204.428358"><ListOfMolecules><Molecule id="S3_M1" name="MT2"><ListOfComponents><Component id="S3_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="S3_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Species><Species id="S4" name="speciesPattern3" concentration="0.0"><ListOfMolecules><Molecule id="S4_M1" name="MT3"><ListOfComponents><Component id="S4_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="S4_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Species><Species id="S5" name="speciesPattern4" concentration="0.0"><ListOfMolecules><Molecule id="S5_M1" name="MT4"><ListOfComponents><Component id="S5_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="S5_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Species></ListOfSpecies><ListOfReactionRules><ReactionRule id="RR1" name="r00" symmetry_factor="1.0"><ListOfReactantPatterns><ReactantPattern id="RR1_RP1"><ListOfMolecules><Molecule id="RR1_RP1_M1" name="MT0"><ListOfComponents><Component id="RR1_RP1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="RR1_RP1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></ReactantPattern><ReactantPattern id="RR1_RP2"><ListOfMolecules><Molecule id="RR1_RP2_M1" name="MT2"><ListOfComponents><Component id="RR1_RP2_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="RR1_RP2_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></ReactantPattern></ListOfReactantPatterns><ListOfProductPatterns><ProductPattern id="RR1_PP1"><ListOfMolecules><Molecule id="RR1_PP1_M1" name="MT1"><ListOfComponents><Component id="RR1_PP1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="RR1_PP1_M1_C2" name="AAB" state="1" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></ProductPattern></ListOfProductPatterns><Map><MapItem sourceID="RR1_RP1_M1" /><MapItem sourceID="RR1_RP1_M1_C1" /><MapItem sourceID="RR1_RP1_M1_C2" /><MapItem sourceID="RR1_RP2_M1" /><MapItem sourceID="RR1_RP2_M1_C1" /><MapItem sourceID="RR1_RP2_M1_C2" /></Map><ListOfOperations><Add id="RR1_PP1_M1" /><Delete id="RR1_RP1" DeleteMolecules="0" /><Delete id="RR1_RP2" DeleteMolecules="0" /></ListOfOperations><RateLaw id="RR1" type="Ele" totalrate="0"><ListOfRateConstants><RateConstant value="K_reserved_0" /></ListOfRateConstants></RateLaw></ReactionRule><ReactionRule id="RR2" name="r00_reverse" symmetry_factor="1.0"><ListOfReactantPatterns><ReactantPattern id="RR2_RP1"><ListOfMolecules><Molecule id="RR2_RP1_M1" name="MT1"><ListOfComponents><Component id="RR2_RP1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="RR2_RP1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></ReactantPattern></ListOfReactantPatterns><ListOfProductPatterns><ProductPattern id="RR2_PP1"><ListOfMolecules><Molecule id="RR2_PP1_M1" name="MT0"><ListOfComponents><Component id="RR2_PP1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="RR2_PP1_M1_C2" name="AAB" state="1" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></ProductPattern><ProductPattern id="RR2_PP2"><ListOfMolecules><Molecule id="RR2_PP2_M1" name="MT2"><ListOfComponents><Component id="RR2_PP2_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="RR2_PP2_M1_C2" name="AAB" state="1" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></ProductPattern></ListOfProductPatterns><Map><MapItem sourceID="RR2_RP1_M1" /><MapItem sourceID="RR2_RP1_M1_C1" /><MapItem sourceID="RR2_RP1_M1_C2" /></Map><ListOfOperations><Add id="RR2_PP1_M1" /><Add id="RR2_PP2_M1" /><Delete id="RR2_RP1" DeleteMolecules="0" /></ListOfOperations><RateLaw id="RR2" type="Ele" totalrate="0"><ListOfRateConstants><RateConstant value="K_reserved_1" /></ListOfRateConstants></RateLaw></ReactionRule><ReactionRule id="RR3" name="r10" symmetry_factor="1.0"><ListOfReactantPatterns><ReactantPattern id="RR3_RP1"><ListOfMolecules><Molecule id="RR3_RP1_M1" name="MT3"><ListOfComponents><Component id="RR3_RP1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="RR3_RP1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></ReactantPattern></ListOfReactantPatterns><ListOfProductPatterns><ProductPattern id="RR3_PP1"><ListOfMolecules><Molecule id="RR3_PP1_M1" name="MT4"><ListOfComponents><Component id="RR3_PP1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="RR3_PP1_M1_C2" name="AAB" state="1" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></ProductPattern></ListOfProductPatterns><Map><MapItem sourceID="RR3_RP1_M1" /><MapItem sourceID="RR3_RP1_M1_C1" /><MapItem sourceID="RR3_RP1_M1_C2" /></Map><ListOfOperations><Add id="RR3_PP1_M1" /><Delete id="RR3_RP1" DeleteMolecules="0" /></ListOfOperations><RateLaw id="RR3" type="Ele" totalrate="0"><ListOfRateConstants><RateConstant value="K_reserved_2" /></ListOfRateConstants></RateLaw></ReactionRule><ReactionRule id="RR4" name="r10_reverse" symmetry_factor="1.0"><ListOfReactantPatterns><ReactantPattern id="RR4_RP1"><ListOfMolecules><Molecule id="RR4_RP1_M1" name="MT4"><ListOfComponents><Component id="RR4_RP1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="RR4_RP1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></ReactantPattern></ListOfReactantPatterns><ListOfProductPatterns><ProductPattern id="RR4_PP1"><ListOfMolecules><Molecule id="RR4_PP1_M1" name="MT3"><ListOfComponents><Component id="RR4_PP1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="RR4_PP1_M1_C2" name="AAB" state="1" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></ProductPattern></ListOfProductPatterns><Map><MapItem sourceID="RR4_RP1_M1" /><MapItem sourceID="RR4_RP1_M1_C1" /><MapItem sourceID="RR4_RP1_M1_C2" /></Map><ListOfOperations><Add id="RR4_PP1_M1" /><Delete id="RR4_RP1" DeleteMolecules="0" /></ListOfOperations><RateLaw id="RR4" type="Ele" totalrate="0"><ListOfRateConstants><RateConstant value="K_reserved_3" /></ListOfRateConstants></RateLaw></ReactionRule></ListOfReactionRules><ListOfObservables><Observable id="O1" name="O0_MT0_tot_Count" type="Molecules"><ListOfPatterns><Pattern id="O1_P1"><ListOfMolecules><Molecule id="O1_P1_M1" name="MT0"><ListOfComponents><Component id="O1_P1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="O1_P1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Pattern></ListOfPatterns></Observable><Observable id="O2" name="O0_MT1_tot_Count" type="Molecules"><ListOfPatterns><Pattern id="O2_P1"><ListOfMolecules><Molecule id="O2_P1_M1" name="MT1"><ListOfComponents><Component id="O2_P1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="O2_P1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Pattern></ListOfPatterns></Observable><Observable id="O3" name="O0_MT2_tot_Count" type="Molecules"><ListOfPatterns><Pattern id="O3_P1"><ListOfMolecules><Molecule id="O3_P1_M1" name="MT2"><ListOfComponents><Component id="O3_P1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="O3_P1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Pattern></ListOfPatterns></Observable><Observable id="O4" name="O0_MT3_tot_Count" type="Molecules"><ListOfPatterns><Pattern id="O4_P1"><ListOfMolecules><Molecule id="O4_P1_M1" name="MT3"><ListOfComponents><Component id="O4_P1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="O4_P1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Pattern></ListOfPatterns></Observable><Observable id="O5" name="O0_MT4_tot_Count" type="Molecules"><ListOfPatterns><Pattern id="O5_P1"><ListOfMolecules><Molecule id="O5_P1_M1" name="MT4"><ListOfComponents><Component id="O5_P1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="O5_P1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Pattern></ListOfPatterns></Observable><Observable id="O6" name="s0_Count" type="Species"><ListOfPatterns><Pattern id="O6_P1"><ListOfMolecules><Molecule id="O6_P1_M1" name="MT0"><ListOfComponents><Component id="O6_P1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="O6_P1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Pattern></ListOfPatterns></Observable><Observable id="O7" name="s1_Count" type="Species"><ListOfPatterns><Pattern id="O7_P1"><ListOfMolecules><Molecule id="O7_P1_M1" name="MT1"><ListOfComponents><Component id="O7_P1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="O7_P1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Pattern></ListOfPatterns></Observable><Observable id="O8" name="s2_Count" type="Species"><ListOfPatterns><Pattern id="O8_P1"><ListOfMolecules><Molecule id="O8_P1_M1" name="MT2"><ListOfComponents><Component id="O8_P1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="O8_P1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Pattern></ListOfPatterns></Observable><Observable id="O9" name="s3_Count" type="Species"><ListOfPatterns><Pattern id="O9_P1"><ListOfMolecules><Molecule id="O9_P1_M1" name="MT3"><ListOfComponents><Component id="O9_P1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="O9_P1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Pattern></ListOfPatterns></Observable><Observable id="O10" name="s4_Count" type="Species"><ListOfPatterns><Pattern id="O10_P1"><ListOfMolecules><Molecule id="O10_P1_M1" name="MT4"><ListOfComponents><Component id="O10_P1_M1_C1" name="AAA" state="c0" numberOfBonds="0" /><Component id="O10_P1_M1_C2" name="AAB" state="0" numberOfBonds="0" /></ListOfComponents></Molecule></ListOfMolecules></Pattern></ListOfPatterns></Observable></ListOfObservables><ListOfFunctions><Function id="O0_MT0_tot"><ListOfReferences><Reference name="UnitFactor_uM_um3_molecules_neg_1" type="ConstantExpression" /><Reference name="Size_c0" type="ConstantExpression" /><Reference name="O0_MT0_tot_Count" type="Observable" /></ListOfReferences><Expression>((O0_MT0_tot_Count * UnitFactor_uM_um3_molecules_neg_1) / Size_c0)</Expression></Function><Function id="O0_MT1_tot"><ListOfReferences><Reference name="O0_MT1_tot_Count" type="Observable" /><Reference name="UnitFactor_uM_um3_molecules_neg_1" type="ConstantExpression" /><Reference name="Size_c0" type="ConstantExpression" /></ListOfReferences><Expression>((O0_MT1_tot_Count * UnitFactor_uM_um3_molecules_neg_1) / Size_c0)</Expression></Function><Function id="O0_MT2_tot"><ListOfReferences><Reference name="UnitFactor_uM_um3_molecules_neg_1" type="ConstantExpression" /><Reference name="Size_c0" type="ConstantExpression" /><Reference name="O0_MT2_tot_Count" type="Observable" /></ListOfReferences><Expression>((O0_MT2_tot_Count * UnitFactor_uM_um3_molecules_neg_1) / Size_c0)</Expression></Function><Function id="O0_MT3_tot"><ListOfReferences><Reference name="UnitFactor_uM_um3_molecules_neg_1" type="ConstantExpression" /><Reference name="Size_c0" type="ConstantExpression" /><Reference name="O0_MT3_tot_Count" type="Observable" /></ListOfReferences><Expression>((O0_MT3_tot_Count * UnitFactor_uM_um3_molecules_neg_1) / Size_c0)</Expression></Function><Function id="O0_MT4_tot"><ListOfReferences><Reference name="UnitFactor_uM_um3_molecules_neg_1" type="ConstantExpression" /><Reference name="Size_c0" type="ConstantExpression" /><Reference name="O0_MT4_tot_Count" type="Observable" /></ListOfReferences><Expression>((O0_MT4_tot_Count * UnitFactor_uM_um3_molecules_neg_1) / Size_c0)</Expression></Function><Function id="s0"><ListOfReferences><Reference name="UnitFactor_uM_um3_molecules_neg_1" type="ConstantExpression" /><Reference name="Size_c0" type="ConstantExpression" /><Reference name="s0_Count" type="Observable" /></ListOfReferences><Expression>((UnitFactor_uM_um3_molecules_neg_1 * s0_Count) / Size_c0)</Expression></Function><Function id="s1"><ListOfReferences><Reference name="UnitFactor_uM_um3_molecules_neg_1" type="ConstantExpression" /><Reference name="Size_c0" type="ConstantExpression" /><Reference name="s1_Count" type="Observable" /></ListOfReferences><Expression>((UnitFactor_uM_um3_molecules_neg_1 * s1_Count) / Size_c0)</Expression></Function><Function id="s2"><ListOfReferences><Reference name="s2_Count" type="Observable" /><Reference name="UnitFactor_uM_um3_molecules_neg_1" type="ConstantExpression" /><Reference name="Size_c0" type="ConstantExpression" /></ListOfReferences><Expression>((UnitFactor_uM_um3_molecules_neg_1 * s2_Count) / Size_c0)</Expression></Function><Function id="s3"><ListOfReferences><Reference name="UnitFactor_uM_um3_molecules_neg_1" type="ConstantExpression" /><Reference name="Size_c0" type="ConstantExpression" /><Reference name="s3_Count" type="Observable" /></ListOfReferences><Expression>((UnitFactor_uM_um3_molecules_neg_1 * s3_Count) / Size_c0)</Expression></Function><Function id="s4"><ListOfReferences><Reference name="UnitFactor_uM_um3_molecules_neg_1" type="ConstantExpression" /><Reference name="Size_c0" type="ConstantExpression" /><Reference name="s4_Count" type="Observable" /></ListOfReferences><Expression>((UnitFactor_uM_um3_molecules_neg_1 * s4_Count) / Size_c0)</Expression></Function></ListOfFunctions></model><jms><broker>k8s-wn-01.cam.uchc.edu:30163</broker><jmsUser>clientUser</jmsUser><pw>dummy</pw><queue>workerEvent</queue><topic>serviceControl</topic><vcellUser>schaff</vcellUser><simKey>273069657</simKey><jobIndex>0</jobIndex></jms></sbml>