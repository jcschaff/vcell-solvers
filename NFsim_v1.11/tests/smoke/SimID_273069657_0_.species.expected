# nfsim generated species list for system: 'nameless'
# warning! this feature is not yet fully tested! 
MT0(AAA~c0,AAB~0)  532
MT1(AAA~c0,AAB~0)  973
MT2(AAA~c0,AAB~0)  1134
